src/Discord.Net.Commands/Extensions/MessageExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static bool HasStringPrefix(this IUserMessage msg, string str, ref int ar
         public static bool HasMentionPrefix(this IUserMessage msg, IUser user, ref int argPos)
         {
             var text = msg.Content;
-            if (text.Length <= 3 || text[0] != '<' || text[1] != '@') return false;
+            if (!string.IsNullOrEmpty(text) && text.Length <= 3 || text[0] != '<' || text[1] != '@') return false;
```

Comment:
These parameters should be grouped together.
```suggestion
            if (!string.IsNullOrEmpty(text) && (text.Length <= 3 || text[0] != '<' || text[1] != '@')) return false;
```

Reply:
Done.